app/controllers/groups_controller.rb

```diff
@@ -466,12 +477,15 @@ def handle_membership_request
       GroupRequest.where(group_id: group.id, user_id: user.id).delete_all
     end
 
+    # do we want to send this reply in the requesting user's language as well?
```

Comment:
We are sending the reply in the user's locale now, so this comment is no longer needed. Right?